api/routers/metrics/metrics.py

```diff
@@ -37,7 +38,7 @@ def get_cameras_for_areas(areas: Iterator[str]) -> Iterator[str]:
     cameras = []
     for area_config in config.values():
         if area_config["Id"] in areas:
-            cameras.extend(area_config[CAMERAS].split(","))
+            cameras.extend(area_config["Cameras"].split(","))
```

Comment:
This was fixing on a separate PR and probably is causing conflict, I changed it to CAMERAS.capitalize()